manifests/transport_https.pp

```diff
@@ -0,0 +1,13 @@
+# Allow APT to use repos served over HTTPS
+class apt::transport_https (
+  $ensure = 'present'
+) {
+  if $ensure !~ /^(present|absent|purged|held|latest)$/ {
+    fail('ensure must be either present, absent, purged, held, or latest')
+  }
+
+  package { ['apt-transport-https', 'ca-certificates']:
+    ensure => $ensure,
+    before => Apt::Setting<| |>
```

Comment:
I suppose this would interfere if people do `Class['apt::update'] -> Package <||>`. Opinions?

Also, I'm not sure how to test this.

And, it's breaking Travis.